samples/MvcAngular/AngularClient/.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+	"directory" : "app/dist"
```

Comment:
Not sure a line break is needed, but you should use spaces instead of tabs :smile: